src/main/java/org/apache/mesos/offer/TaskUtils.java

```diff
@@ -18,6 +21,8 @@
  */
 public class TaskUtils {
     private static final Logger LOGGER = LoggerFactory.getLogger(TaskUtils.class);
+    private static final int CONFIG_TEMPLATE_LIMIT_BYTES = 100 * 1024; // 100KB
```

Comment:
I don't know why this concerns me, but this is a very small not configurable number.  I don't think we should make it configurable, at least not yet.  Maybe bump this to 500KB or something?  I have no data to back up this concern.  Feel free to ignore.